### PR TITLE
Tonguespike buff, chemspike slight nerf

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Increases the tonguespike damage.
impact damage: 15 -> 25
pain damage (leaving tonguespike embedded damage): 4 -> 15
cooldown time = 10 seconds -> 1 seconds

Decreases the chemspike damage
impact damage: 4? -> 2
pain damage: 4 -> 0

## Why It's Good For The Game

These values were eyeballed when I originally added them, fuck embedding code, uuuh... these values make more sense. At the very least tonguespike was nearly unusuable being a 15 damage item that pain-ticked for a pitiful 4.
The cooldown was just unnecessary and killed unnecessary organ regeneration strategies

Chemspike ticked for too much when it wasn't really meant to deal much damage.

## Changelog
:cl:
balance: Tonguespike is better, chemspike deals slightly less damage
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
i do not understand why anyone would ever use these mutations

**Comment by tralezab** [comment]
> i do not understand why anyone would ever use these mutations

lings can regen their tongue, chemspike can remotely transfer a fuckton of reagents into someone

**Comment by tralezab** [comment]
that being said, would be nice for these to be less bad and would take suggestions... only problem is that if it's too good, genetics as a system will mean everyone can have it and then it needs to get nerfed again. fuckin' genetics. would work better as a dna mutant ability

**Comment by Overuseddart** [comment]
> that being said, would be nice for these to be less bad and would take suggestions... only problem is that if it's too good, genetics as a system will mean everyone can have it and then it needs to get nerfed again. fuckin' genetics. would work better as a dna mutant ability

Maybe like an effect that makes it easier for the person who shot the tounge to more easily follow up?
Buffing the attacker specifically against the guy who got hit would make it better and not just a tiny bit of damage.

**Comment by carlarctg** [comment]
> that being said, would be nice for these to be less bad and would take suggestions... only problem is that if it's too good, genetics as a system will mean everyone can have it and then it needs to get nerfed again. fuckin' genetics. would work better as a dna mutant ability

let em attach the tongue back

**Comment by atlasle** [comment]
> let em attach the tongue back

actually make it damage the tongue, be unable to be used with a severely damaged BUT it also lets you reattach any tongue by holding one and clicking while targeting the mouth (action would take like 3-5 seconds though)

File: code/modules/projectiles/projectile.dm [hunk 1]
```diff
@@ -1061,6 +1061,7 @@
 	if(!shrapnel_type || !LAZYLEN(embedding))
 		return
 
+	// sigh
```
Comment (ArcaneDefence) [inline code/modules/projectiles/projectile.dm:1064]: Why touch projectile.dm just to add a comment that says sigh

Comment (tralezab) [reply code/modules/projectiles/projectile.dm:1064]: bro did you see the line below it

Comment (ArcaneDefence) [reply code/modules/projectiles/projectile.dm:1064]: It's still cringe as fuck to do nothing, explain nothing, and touch a file to explicitly state that nothing.

**Comment by moocowswag** [comment]
> > i do not understand why anyone would ever use these mutations
> 
> lings can regen their tongue, chemspike can remotely transfer a fuckton of reagents into someone

augmented people can just put a new tongue back in.

**Comment by moocowswag** [comment]
> It's still not an aimed spell, so if you're not standing in a cardinal direction of the shooter you literally cannot be hit by it smadge The biggest issue in its use is that the experience of using it is dogshit, regardless of the numbers being funny or not

firebreath suffers the same issue, and i dont like the trend genetics is going for stuff that is balanced by i wouldnt say unwieldy i would say anti qol.

File: code/modules/projectiles/projectile.dm (hunk 1 above)
Comment (tralezab) [reply code/modules/projectiles/projectile.dm]: removed your highness

**Comment by JohnFulpWillard** [comment]
>  i dont like the trend genetics is going for stuff that is balanced by..... i wouldnt say unwieldy, i would say anti qol.

**Comment by tralezab** [comment]
> > It's still not an aimed spell, so if you're not standing in a cardinal direction of the shooter you literally cannot be hit by it smadge The biggest issue in its use is that the experience of using it is dogshit, regardless of the numbers being funny or not
> 
> firebreath suffers the same issue

All of genetics suffers the same issue of it being too good = nerf because good mutations can quickly spread to the entire station. If you want genetics powers to be good, you have to start by making genetics powers not free handout once discovered

**Comment by JohnFulpWillard** [comment]
>  i dont like the trend genetics is going for stuff that is balanced by..... i wouldnt say unwieldy, i would say anti qol.

This is literally what Genetics was from the very start.

Regardless, Armhulenn has said that it was technical issues back when he originally added it, that prevented this from being a pointed spell. You can feel free to change that now.

**Comment by moocowswag** [comment]
> > > It's still not an aimed spell, so if you're not standing in a cardinal direction of the shooter you literally cannot be hit by it smadge The biggest issue in its use is that the experience of using it is dogshit, regardless of the numbers being funny or not
> > 
> > 
> > firebreath suffers the same issue
> 
> All of genetics suffers the same issue of it being too good = nerf because good mutations can quickly spread to the entire station. If you want genetics powers to be good, you have to start by making genetics powers not free handout once discovered

Nobody is asking for the spells to be strong we just want them to not be unsatisfying / annoying to use unless thats the power`s gimmick. you can never a spell in numbers or mechanics to make it more usable.

**Comment by tralezab** [comment]
I can add tonguespike improvements like that, sure

**Comment by moocowswag** [comment]
Might i also suggest adding a power chromosome interactions to this spell? If not I might do it in the future then nerf some of the base stats of tongue spike so its not op with power boost.

**Pull request merged**